themes/timer-hugo/layouts/allpublications/list.html

```diff
@@ -0,0 +1,24 @@
+{{ define "main" }}
+
+{{ partial "page-title.html" . }}
+
+{{ if .Site.Data.list_publications.enable }}
+{{ "<!-- Complete List Publications Section Start -->" | safeHTML }}
+<section id="publications">
+    <div class="container">
+        <div class="section-heading" style="margin-top:30px;margin-bottom:20px">
+            {{ with .Site.Data.list_publications.title }}<h2 style="text-align:center">{{ . }}</h2>{{ end }}
+        </div>
+        <ol style="text-align:justify">
```

Comment:
```suggestion
        <ol style="text-align:justify" reversed>
```

So the numbering start at the total amount of papers.